dotnet/samples/README.md

```diff
@@ -1 +1,121 @@
+
+2. Checkout the `DOTNET` branch.
+
+3. Navigate to the `teams-ai/dotnet/packages/Microsoft.TeamsAI` folder.
+
+4. Run `dotnet pack` in terminal.
+
+5. You should then see the following output: 
+
+`Successfully created package "C:...\teams-ai\dotnet\packages\Microsoft.TeamsAI\Microsoft.TeamsAI\bin\Debug\Microsoft.TeamsAI.1.0.0.nupkg"`
+
+6. Move the `Microsoft.TeamsAI.1.0.0.nupkg` to the `LocalPkg/` folder within the sample folder.
+
+7. Now you can install the `Microsoft.TeamsAI` package from the Visual Studio's `Manage NuGet Packages` flow or by running `dotnet add Microsoft.TeamsAI`.
+
+Now can you proceed to setting up the sample.
+
+## Setting up a sample
+
+1. Clone the repository:
+
+`git clone https://github.com/Microsoft/teams-ai.git`
+
+2. Navigate to the `teams-ai/dotnet/samples` folder, pick a sample (ex. `01.a.echoBot`) and open the `.sln` file. 
+
+By this point you should have your sample open in your IDE of choice.
+
+There are a few ways to get the application up and running. The latest way is using Teams toolkit with Visual Studio. However you can also set it up manually. You can find instructions for both below:
+
+### Using Teams Toolkit for Visual Studio
+
+#### Additional Prerequisites
+
+- [Visual Studio 2022 17.7.0 Preview 3.0](https://visualstudio.microsoft.com/vs/preview/)
+- Teams Toolkit extension (installed through the Visual Studio installer)
+  - Select Microsoft Teams development tools under ASP.NET and web development.
+  - ![Teams Toolkit Installation](/dotnet/samples/assets/ttk-install.png)
+
+#### Steps
+1. Open the solution in Visual Studio. (For example `EchoBot.sln`).
+1. In the debug dropdown menu, select `Dev Tunnels > Create A Tunnel` (set authentication type to Public) or select an existing public dev tunnel
+2. Right-click your project and select `Teams Toolkit > Prepare Teams App Dependencies`
+3. If prompted, sign in with a Microsoft 365 account for the Teams organization you want 
+to install the app to. 
+
+
+>If you do not have permission to upload custom apps (sideloading), Teams Toolkit will 
+recommend creating and using a [Microsoft 365 Developer Program](https://developer.microsoft.com/en-us/microsoft-365/dev-program) account - 
+a free program to get your own dev environment sandbox that includes Teams.
+
+4. Press F5, or select the `Debug > Start` Debugging menu in Visual Studio
+5. In the launched browser, select the Add button to load the app in Teams
+6. This should redirect you to a chat window with the bot. 
+
+### Manually upload the app to a Teams desktop client
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because the Teams service needs to call into the bot.
+
+#### Additional requirements
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+#### Steps
+1) Run ngrok - point to port 5130
+
+    ```bash
+    ngrok http 5130 --host-header="localhost:5130"
+    ```
+
+1) Setup for Bot
```

Comment:
I know this is from legacy documentation, but to me it's vague what the user will be learning. Is there a different phrase we can put here?

* I'll be sure to update the `js` side accordingly